test/integration/sequelize.test.js

```diff
@@ -384,6 +384,67 @@ describe(Support.getTestDialectTeaser('Sequelize'), () => {
   });
 
   describe('truncate', () => {
+    it('truncates all models sequentially in cascade', async function () {
+      const Project = this.sequelize.define(`project${config.rand()}`, {
+        id: {
+          type: DataTypes.INTEGER,
+          primaryKey: true,
+          autoIncrement: true
+        },
+        title: DataTypes.STRING
+      });
+      const Project2 = this.sequelize.define(`project2${config.rand()}`, {
+        id: {
+          type: DataTypes.INTEGER,
+          primaryKey: true,
+          autoIncrement: true
+        },
+        title: DataTypes.STRING
+      });
+      const TopProject = this.sequelize.define(`top-project${config.rand()}`, {
+        id: {
+          type: DataTypes.INTEGER,
+          primaryKey: true,
+          autoIncrement: true
+        },
+        title: DataTypes.STRING
+      });
```

Comment:
I am 99% sure the following simplification should still work, can you please check?

```suggestion
      const Project = this.sequelize.define(`project-${config.rand()}`, { title: DataTypes.STRING });
      const Project2 = this.sequelize.define(`project2-${config.rand()}`, { title: DataTypes.STRING });
      const TopProject = this.sequelize.define(`top-project-${config.rand()}`, { title: DataTypes.STRING });
```